### ISSUE TITLE
Legitimate emails getting flagged as spam

### ISSUE THREAD

**Issue by LukeChung** [MicrosoftDocs/microsoft-365-docs#832]
Our legitimate emails are getting flagged as spam. It's surprising since email contains customer specific information so it's not the same email for each recipient. How do we turn this feature off or flag our emails as legitimate so this policy knows that we've approved them and it's not some rogue process?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c3c27660-0f0b-3002-7183-854e4fb8716b
* Version Independent ID: 3f436862-e6b3-2a9d-70b5-f7eee02eec2e
* Content: [Configure the outbound spam policy](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/configure-the-outbound-spam-policy)
* Content Source: [microsoft-365/security/office-365-security/configure-the-outbound-spam-policy.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/configure-the-outbound-spam-policy.md)
* Service: **o365-seccomp**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

**Comment by mypil** [comment]
@LukeChung - Thank you for submitting feedback.

I think the best way forward is if you can open a **[service ticket](https://docs.microsoft.com/office365/admin/contact-support-for-business-products)** in your tenant so this can get resolved ASAP. Based on outcome, let me know if it is something that can be called out in the docs.

If you don't mind please keep us posted here on the resolution and I really want to get this information you are discovering back into the docs.

**Comment by LukeChung** [comment]
We've submitted a ticket to no avail. The problem is the algorithm for flagging outbound spam is not good enough to differentiate between legitimate emails that may go out to a group of contacts vs. a real spammer, and can't be turned off. Having it trigger a blockage after 100 emails is way too low a threshold. Worse, it is not going to stop real spammers who do a lot more than 100 at a time. 

This is especially annoying as we had to notify all of our customers of a bug Microsoft introduced with a Security Update (http://fmsinc.com/MicrosoftAccess/Errors/query_is_corrupt/), and Office 365 is preventing us from doing so. It just makes a bad situation worse. If a message is flagged as spam, Office 365 should allow customers to approve it and continue emailing it. If the customer is later found to be spamming, that can be dealt with but customers shouldn't be treated as guilty in advance. Note, we are well within the daily limits of Office 365 email accounts. It's the content of the emails that's being judged.

There are many situations with alerts where the failure to send out timely emails can lead to serious harm both financially and bodily. The change needs to be made at the policy level, or an explanation for how we need to change our emails to avoid having them flagged as spam. 

Note, the last time we had a similar need was in March when we broadcasted without any blockage.
